lib/operations/to_array.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const Aspect = require('./operation').Aspect;
+const defineAspects = require('./operation').defineAspects;
+const handleCallback = require('../utils').handleCallback;
+const OperationBase = require('./operation').OperationBase;
+const push = Array.prototype.push;
+
+let cursor;
+function loadCursor() {
```

Comment:
How many copies of `loadCursor()` do we have now? I know that we had to roll back our original change that broke webpack, but could we do something like making a single file `dynamic_loaders.js` (or stick it in `utils.js`)?

```js
// lib/dynamic_loaders.js
let cursor;
let db;
let collection;

function loadCursor() {
  if (!cursor) {
    cursor = require('./cursor');
  }
  return cursor;
}

function loadDb() {
  if (!db) {
    db = require('./db');
  }
  return db;
}

function loadCollection() {
  if (!collection) {
    collection = require('./collection');
  }
  return collection;
}

module.exports = { loadCursor, loadDb, loadCollection }; 

// lib/operations.to_array.js

const loadCursor = require('../dynamic_loaders').loadCursor;
...
```